Pipfile

```diff
@@ -15,9 +15,10 @@ bcrypt = "==3.1.7"
 django-extensions = "==2.2.5"
 pydotplus = "==2.0.2"
 psycopg2-binary = "*"
-coverage = "==4.5.4"
 Markdown="==3.2.1"
 drf-yasg="==1.17.1"
+django-nose="==1.4.6"
+coverage-badge="==1.0.1"
 
 [requires]
-python_version = "3.7.4"
+python_version = "3.7"
```

Comment:
Is there a reason, why we bump down the version to Python 3.7?

Reply:
pipenv on teamcity was complaining about the version. Doing it this way resolved the issue

Reply:
Nah. If that's so, it's not a big deal